ykrt/pass_finder/try_passes.py

```diff
@@ -77,7 +82,7 @@ def test_pipeline(logf, pl):
     log(logf, "\n\n" + str(pl) + "\n")  
 
     # Make sure we don't run empty strings in pipeline.
-    assert (len(pl.pre_link) != 0 and len(pl.link_time) != 0), "Both prelink and postlink passes cannot be empty!!!"
+    assert (len(pl.pre_link) != 0 or len(pl.link_time) != 0), "Both prelink and postlink passes cannot be empty!!!"
```

Comment:
so we will need this to be an `or` since the assert will fails  and exit with error message only when both pre_link and link_time pipeline are parsed with empty string, otherwise if any one of the prelink or link_time have non-empty string the script should proceed normally.